ModerationPolicy.md

```diff
@@ -0,0 +1,356 @@
+to all members of the Moderation Team.
+
+When a request is sent by email to the [report@expressjs.com][] (or directly to a
+Moderation Team member) the moderation team must log the issue internally and
+report it periodically to the TC.
+
+Requests should contain as much information and context as possible, including
+the URL and a screenshot of the Post in question. Screenshots may be modified
+to obscure obscene or offensive content.
+
+External public venues or social media services such as Twitter must never be
+used to request Moderation.
+
+Collaborators must never discuss the specific details of a Moderation request
+in any public forum or any social media service outside of the expressjs GitHub
+Organizations.
+
+Note that quoting the original content of a Post within a Moderation request or
+expressjs/moderation repository issue is not a violation of the
+[Code of Conduct][]. However, discretion is advised when including such quotes
+in requests posted to public repositories.
+
+Requests for Moderation that do not appear to have been submitted in good faith
+with intent to address a legitimate [Code of Conduct][] violation will be
+ignored.
+
+## Consideration of Intent
+
+Before Moderating a Post, Collaborators should carefully consider the possible
+intent of the author. It may be that the author has simply made an error or is
+not yet familiar with the [Code of Conduct][]; or it may be that cultural
+differences exist, or that the author is unaware that certain content is
+inappropriate. In such cases, the author should be given an
+opportunity to correct any error that may have been made.
+
+Note, however, that unfamiliarity with the [Code of Conduct][] does not excuse
+a Post from Moderation.
+
+## Guidelines and Requirements
+
+-   All Posts are expected to respect the expressjs [Code of Conduct][].
+-   Any Collaborator with triage permission to a given repository (except
+    expressjs/moderation) may Moderate Posts within that repository's issue tracker.
+    Only the Moderation team is allowed to moderate posts on expressjs/moderation.
+-   The Moderation Team serves as the final arbiter for all Moderation issues.
+-   Moderation Team members may Remove or Block an individual from the expressjs
+    GitHub Organizations.
+-   For any Removal or Blocking action, an issue describing the reasons for the
+    action, and identifying the Github account being acted upon, must be posted
+    to the Moderation Repository with an explanation provided by the Moderation
+    Team member performing the action.
+-   Any individual Blocked from the expressjs, jshttp, and pillarjs organizations will be recommended
+    for exclusion from any OpenJS Foundation sponsored event or activity.
+-   Minor edits to the formatting of a Post or to correct typographical errors
+    are not "Moderation". Such edits and their intent must
+    still be documented with a short note indicating who made the edit and why.
```

Comment:
github's "edits" feature already documents who made the edit and what change was made; only the "why" wouldn't be included.

Reply:
i assume that would be the moderation repo. but also i think this will either create a lot of useless noise, or will prevent a lot of helpful corrections, because the "why" will likely be pretty apparent.

Reply:
Rephrasing it to include _why_.

Reply:
Done.